.github/workflows/deploy-arbitrum-goerli-subgraph.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: '16'
```

Comment:
Nitpick, but EoL date for Node 16 is today. It should be safe to bump to 18 or even 20 here.

Reply:
Good catch, wasn't aware of this! We can update all our workflows to use the new versions, I had to bump it to 16 at least so that it actually was able to run, since 14 wasn't working anymore 😅